src/views/ModalView.ts

```diff
@@ -0,0 +1,20 @@
+class ModalView {
+    constructor(modalContent: HTMLElement) {
+        const modal = document.createElement('div');
+        modal.id = 'modal';
+        const content = document.createElement('div');
+        content.className = 'modal-content';
+        content.appendChild(modalContent);
+        modal.appendChild(content);
+        document.body.appendChild(modal);
```

Comment:
konstruktor powinien być krotki i ewentualnie wywoływać metody potrzebne, można to wynieść do metody np setupModal() czy cos